src/components/products/product-list/ko/runtime/product-list.ts

```diff
@@ -97,6 +99,139 @@ export class ProductList {
         }
     }
 
+     //-----pagination changes start ---------------------------- 
```

Comment:
Please remove the comment `//-----pagination changes start ----------------------------`